src/scss/components/_modal-box.scss

```diff
@@ -218,4 +121,54 @@ body.stopPageScroll {
 		width: 100%;
 		object-fit: contain;
 	}
+}
+
+.modal-close-button {
+	position: absolute;
+	width: 16px;
+	height: 16px;
+	top: 8px;
+	right: 8px;
+	background-color: helpers.$color-white;
+
+	@include helpers.mobile() {
+		top: 20px;
+		right: 20px;
+	}
+
+	@include helpers.laptop() {
+		top: 17px;
+		right: 17px;
+	}
+
+	&::after {
+		content: "";
+		position: absolute;
+		width: 200%;
+		height: 200%;
+		left: -50%;
+		top: -50%;
+	}
+
+	.modal-close-button__line {
```

Comment:
&__line{...}

check suh places all over the site

Reply:
Done